com.trollworks.gcs/src/com/trollworks/gcs/utility/Log.java

```diff
@@ -30,6 +31,14 @@
             } catch (Throwable throwable) {
                 error("Unable to redirect log to " + property, throwable);
             }
+        } else if (Platform.isWindows() && System.getenv() != null) {
```

Comment:
I think I wasn’t clear before... something like this instead:
```
} else if (Platform.isWindows()) {
    String localAppData = System.getenv(“LOCALAPPDATA”);
    if (localAppData != null) {
        File logDirectory = new File(localAppData + “\\GCS”;
```

Reply:
Fair point, I realize now after I wrote it that I was calling an empty System.getenv() and would not be properly checking for null. Your change also removes an excess call, and that's reflected in my new commit below.